parity/run.rs

```diff
@@ -258,6 +258,11 @@ fn execute_light_impl(cmd: RunCmd, can_restart: bool, logger: Arc<RotatingLogger
 	let service = light_client::Service::start(config, &spec, fetch, db, cache.clone())
 		.map_err(|e| format!("Error starting light client: {}", e))?;
 	let client = service.client();
+	// Note: uncomment this code if you want to regenerate the JSON of the hardcoded sync feature
+	// in the chain specifications.
+	/*if let Some(hs) = client.read_hardcoded_sync() {
```

Comment:
we could make this a subcommand of some kind

Reply:
generally not a fan of the commented-out code. the CLI is a PITA to work with though :\